typings/riteway.d.ts

```diff
@@ -1 +1,16 @@
-declare module 'riteway'
+declare module 'riteway' {
+  export function describe(label: string, callback: describeCallback): void
+
+  type describeCallback = (should: should) => Promise<void>
+
+  type should = (label?: string) => { assert: assert }
+
+  type assert = (assertion: Assertion) => void
+
+  interface Assertion {
+    given: string
+    should: string
```

Comment:
`should` should be an optional property because will default to the label above in the `should` function.

Reply:
fixed. also added readonly, dunno if that works (is there any reason to ever mutate them?)

Reply:
Nope, can't think of anything